src/services/explore.ts

```diff
@@ -113,3 +113,44 @@ export function createModifications(
         }
     );
 }
+
+/**
+ * Create Filter
+ * @returns {Promise<Response>}
+ */
+// {
+//     "type": "IDENTIFIER_LIST",
+//     "equipmentType": "VOLTAGE_LEVEL",
+//     "filterEquipmentsAttributes": [
+//       {
+//         "equipmentID": "fds"
+//       },
+//       {
+//         "equipmentID": "sf"
+//       },
+//       {
+//         "equipmentID": "fes"
+//       }
+//     ]
+//   }
```

Comment:
You will not maintain this comment if DTO change, it's not useful I think

Reply:
done